src/test/scala/centaur/CallCacheSpec.scala

```diff
@@ -26,6 +28,18 @@ class CallCacheSpec extends FlatSpec with Matchers with ParallelTestExecution {
     }
   }
 
+  "dockerHashNoLookup" should "not lookup docker tag if backend doesn't support docker" in {
```

Comment:
Actually I can't check that because the docker attribute is NOT in the metadata. Which is good I guess since the backend ignored it.
I'm going to change the name of this test to something like "ignoreDockerForCallCachingIfBackendDoesntSupportDocker"